tests/testing.R

```diff
@@ -0,0 +1,50 @@
+sourceEntireFolder <- function(folderName, verbose=FALSE, showWarnings=TRUE) {
```

Comment:
No! This is not how you work within an R package :stuck_out_tongue: If you want a script you can execute to test the newest changes put it in the root directory and start with load_all(). Even better: Simply write a unittest with testthat in the tests directory and regularly execute with Shift+Strg+t in RStudio. The package will be loaded automatically before performing the tests.